msg/versioned/ArmingCheckReply.msg

```diff
@@ -1,34 +1,38 @@
+# Arming check reply.
+#
+# Response to an ArmingCheckRequest message.
+
 uint32 MESSAGE_VERSION = 0
 
-uint64 timestamp # time since system start (microseconds)
+uint64 timestamp # Time since system start [us]
 
-uint8 request_id
-uint8 registration_id
+uint8 request_id # Id of ArmingCheckRequest for which this is a response.
+uint8 registration_id # Id of component emitting this response.
 
-uint8 HEALTH_COMPONENT_INDEX_NONE = 0
+uint8 HEALTH_COMPONENT_INDEX_NONE = 0 # Index of health component for which this response applies.
 
-uint8 health_component_index      # HEALTH_COMPONENT_INDEX_*
-bool health_component_is_present
-bool health_component_warning
-bool health_component_error
+uint8 health_component_index # HEALTH_COMPONENT_INDEX_*
+bool health_component_is_present ?
+bool health_component_warning # ?
+bool health_component_error # ?
 
-bool can_arm_and_run              # whether arming is possible, and if it's a navigation mode, if it can run
+bool can_arm_and_run # True if arming is possible, and for a navigation mode, if it can run.
```

Comment:
So this indicates that the report is that this component is ready to arm.

Where it says "and for a navigation mode, if it can run." - what other kinds of modes are there?

Reply:
Yes the component does not have to a mode, but for example could be an obstacle avoidance, remote ID, ...
For modes, `True` means the mode can be activated.

Reply:
Hmmm. What I think you are saying is that if the component is navigation mode, it indicates whether the component meets all requirements for arming in the mode, or switching to the mode if the vehicle is already armed.
This differs slightly from other components, because you can arm the other components such as avoidance when the vehicle is already.

I've proposed an alternative